src/actions/index.js

```diff
@@ -9,10 +9,18 @@ export const DECREMENT = 'DECREMENT';
 // logic of updating the central Redux store. That
 // is left to the reducer(s).
 
-export const increment = () => {
-  // Fill in this function
+export const increment = count => {
+  console.log(`Before increment: ${count}`)
+  return {
+    type: INCREMENT,
+    payload: count
```

Comment:
Technically, you could get by without using this here since count is maintained on state and the state gets passed with the action to the reducer.